### ISSUE TITLE
Add anchors to golr tables to allow forwarding to specific result (or result set)

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/monarch-legacy#711]
It would be useful to link to a specific row (or result set) in a table, for example, in a similar way github allows one to select a line in a source code [view](https://github.com/geneontology/amigo/blob/master/javascript/lib/amigo/linker.js.tests#L60)

This might require edits to bbop.widgets.live_results.  In that case, this may make more sense in the bbop repo.  Coordinate with @ktlm.

**Comment by jmcmurry** [comment]
Good idea. This would be really great, but a long way off since we don't yet commit to URL persistence even at the tab level.
